mcs/class/System.Private.CoreLib/System.Runtime.InteropServices/Marshal.cs

```diff
@@ -42,17 +42,22 @@ partial class Marshal
 		[MethodImplAttribute (MethodImplOptions.InternalCall)]
 		public extern static void StructureToPtr (object structure, IntPtr ptr, bool fDeleteOld);
 
-		internal static IntPtr AllocBSTR (int length)
-		{
-			throw new NotImplementedException ();
-		}
+		[MethodImplAttribute (MethodImplOptions.InternalCall)]
+		public extern static int GetArrayElementSize (Type type);
```

Comment:
Does .NET Core have this public API? can't find it.

Reply:
Fixed.